pkgs/top-level/all-packages.nix

```
@@ -28170,6 +28170,7 @@ with pkgs;
   netcoredbg = callPackage ../development/tools/misc/netcoredbg { };
 
   ncdu = callPackage ../tools/misc/ncdu { };
+  ncdu2 = callPackage ../tools/misc/ncdu2 { };
```

```suggestion
  ncdu_2 = callPackage ../tools/misc/ncdu/2 { };
```
Unless the program is actually named ncdu2, the verison number should be sperated by underscore IIRC.